rpc/src/v1/helpers/errors.rs

```diff
@@ -422,6 +423,19 @@ pub fn filter_not_found() -> Error {
 	}
 }
 
+pub fn filter_block_not_found(id: BlockId) -> Error {
+	Error {
+		code: ErrorCode::ServerError(codes::UNSUPPORTED_REQUEST),
```

Comment:
Is `UNSUPPORTED_REQUEST` the correct code here? Is that the `-32000` from the EIP? If yes, maybe add a comment to that effect.

Reply:
We use `UNSUPPORTED_REQUEST` for other params-invalid errors, like the above `filter_not_found`. So I think it may be fine. :)

Reply:
Ok, you know best! It sounds a little like `HTTP 501 Not Implemented` but is more like `HTTP 400 Bad Request`, hence my confusion.

Reply:
It will return the correct error code https://github.com/paritytech/parity-ethereum/blob/master/rpc/src/v1/helpers/errors.rs#L31.

Reply:
For reference: https://www.jsonrpc.org/historical/json-rpc-over-http.html#errors